Userland/Libraries/LibWeb/DOM/Node.cpp

```diff
@@ -2367,6 +2364,15 @@ ErrorOr<String> Node::name_or_description(NameOrDescription target, Document con
 
                 return IterationDecision::Continue;
             });
+            // ii. Check for CSS generated textual content associated with the current node and include it in the accumulated text. The CSS :before and :after pseudo elements [CSS2] can provide textual content for elements that have a content model.
+            // - For :after pseudo elements, User agents MUST append CSS textual content,
+            // without a space, to the textual content of the current node.
```

Comment:
It feels a bit messy to me to have this spec step duplicated - I'd be inclined to leave it all together above, with a couple of `// NOTE` comments explaining that the code for `::after` is down here.